src/extension.ts

```diff
@@ -176,3 +190,43 @@ async function showUpdateNotification(context: vscode.ExtensionContext) {
     }
   }
 }
+
+function registerStackReportCommands(context:vscode.ExtensionContext) {
+  
+  let disposableFullStackStatusBar = vscode.commands.registerCommand(

```

Comment:
Use const for everywhere, only use let if your variables are changing

Reply:
ack